KOGITO-5553:Data-index API connection with runtimes (#3878)
kiegroup/kie-docs: doc-content/kogito-docs/src/main/asciidoc/release-notes/ReleaseNotesKogito.1.11.0.Final/ReleaseNotesKogito.1.11.0.Final.adoc

Starting from {PRODUCT} 1.11.0, you must upgrade the Infinispan version to 12.1.7 or later to prevent the issues regarding results returned in paginated queries.

=== {PRODUCT} Data Index service supports new Gateway API

A new set of GraphQL queries and mutations are supported to perform the following operations on different {PRODUCT} services:

* Abort a process instance
* Retry a process instance
* Skip a process instance

> For more information, see xref:proc-data-index-service-runtime-connection_kogito-configuring[].

Reviewer: ```suggestion
For more information, see {URL_CONFIGURING_KOGITO}#con-data-index-service_kogito-configuring[_{PRODUCT} Data Index Service_].
```

> === {PRODUCT} Data Index service API queries extended to get data from {PRODUCT} runtime services

Reviewer: ```suggestion
=== {PRODUCT} Data Index service API queries extended to retrieve data from {PRODUCT} runtime services
```

> The queries also have been extended to retrieve data from runtimes {PRODUCT} services, allowing get:

Reviewer: ```suggestion
The {PRODUCT} Data Index service API queries are now extended to retrieve process instance diagram and process instance nodes from runtimes {PRODUCT} services. For more information, see {URL_CONFIGURING_KOGITO}#con-data-index-service_kogito-configuring[_{PRODUCT} Data Index Service_]. 
```

> - The process instance diagram

Reviewer: ```suggestion

```

> - The process instance available Nodes from the process definition

Reviewer: ```suggestion

```

Starting from {PRODUCT} 1.11.0, you must upgrade the Infinispan version to 12.1.7 or later to prevent the issues regarding results returned in paginated queries.

==== {PRODUCT} Data Index service incorporated new API To connect with other {PRODUCT} services

New GraphQL queries and mutations have been added to allow performing operations in different {PRODUCT} services:
The new operations added are:

- Abort process Instance
- Retry process Instance
- Skip process Instance
- Update process Instance variables
- Trigger Node Instance related with a process Instance
- Retrigger an existing Node Instance

> - Cancel a Node instance

Reviewer 2: would probably be worth linking to the new chapter you wrote from here, something like: for more details , visit the Gateway API documentation, etc

Author: added!

> The queries also have been extended to retrieve from runtimes {PRODUCT} services:

Author: done!
